content/_data/changelogs/lts.yml

```diff
@@ -13454,6 +13454,33 @@
       Any <code>RootAction</code> that uses jelly to show/hide is correctly hidden in the header.
       Any <code>RootAction</code> with sub tasks now shows the subtasks as a dropdown when in the correct context.
 
+- version: "2.516.2"
+  date: 2025-08-20
+  changes:
+  - type: bug
+    category: bug
+    pull: 10715
+    issue: 75637
+    authors:
+      - janfaracik
+    pr_title: "[JENKINS-75637] Fix System configuration page no longer having breadcrumb navigation"
+    message: |-
+      Restore breadcrumb navigation for the <strong>System</strong> configuration page.
+  - type: bug
+    category: bug
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-75931] Fix links to anchors"
+    references:
+      - url: https://github.com/jenkinsci/jellydoc-maven-plugin/pull/154
+        title: jellydoc-maven-plugin PR 154
+      - pull: 10928
+      - issue: 75931
+      - url: https://github.com/jenkinsci/stapler-maven-plugin/pull/155
+        title: stapler-maven-plugin PR 155
+    message: |-
+      Fix anchor links in Jenkins Taglib documentation.
+
```

Comment:
Looks like your source branch is not up to date with the `jenkins` remote 's `master` branch.